packages/revai/README.md

```diff
@@ -0,0 +1,38 @@
+# AI SDK - Rev.ai Provider
+
+The **[Rev.ai provider](https://sdk.vercel.ai/providers/ai-sdk-providers/revai)** for the [AI SDK](https://sdk.vercel.ai/docs)
+contains language model support for the Rev.ai transcription API.
+
+## Setup
+
+The Rev.ai provider is available in the `@ai-sdk/revai` module. You can install it with
+
+```bash
+npm i @ai-sdk/revai
+```
+
+## Provider Instance
+
+You can import the default provider instance `revai` from `@ai-sdk/revai`:
+
+```ts
+import { revai } from '@ai-sdk/revai';
+```
+
+## Example
+
+```ts
+import { revai } from '@ai-sdk/revai';
+import { experimental_transcribe as transcribe } from 'ai';
+
+const { text } = await transcribe({
+  model: revai.transcription('machine'),
+  audio: new URL(
+    'https://github.com/vercel/ai/raw/refs/heads/main/examples/ai-core/data/galileo.mp3',
+  ),
```

Comment:
out of curiosity, what happens if the url is entered as a string? (should ideally be supported)

Reply:
(no need for PR updates re this comment)

Reply:
Likely explodes, but I can add string-url support on a new PR!

Reply:
we have special conversion mechanisms that we should be using, no need to this build one-off, and we can also delay and do it on v5 once all providers are there to reduce # of ports.

Reply:
(the mechanism has been introduced / updated on v5 recently)